src/UserModule/dto/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from './new-user.dto';
-export * from './user-update.dto';
-export * from './user.dto';
+export { NewUserDTO } from './new-user.dto';
+export { UserUpdateDTO } from './user-update.dto';
+export { UserDTO } from './user.dto';
```

Comment:
Não entendi porque 3 DTOs. Apenas um não atenderia?

Reply:
Eu separei em 3 porque cada endpoint precisa de uma informação diferente. Todos são parecidos com User, porém tem menos informações né

Reply:
Pelo que vi 3 tem os mesmos campos, porém algumas validações diferentes. Veja se isso é necessário de fato, senão pode trazer uma complexidade a mais pro pessoal, deixar isso só onde de fato for bem diferente.

Reply:
É que usar o mesmo influência até na documentação gerada pelo Swagger. Deixa o mesmo implica que na documentação o front tem que mandar “id”, sendo que eles não tem que mandar, entendeu?